rails-routes.el

```diff
@@ -68,38 +83,44 @@
   (message "Fetching routes.  Please wait.")
   (let ((command-result (cl-remove-if-not
                          (lambda (element)
-                           (or (eq (length (split-string element " +")) 5)
-                               (eq (length (split-string element " +")) 4)))
+                           (let ((len (length (split-string element " +"))))
+                             (or (eq len 5) (eq len 4))))
                          (split-string (shell-command-to-string rails-routes-search-command) "\n"))))
 
-    (if rails-routes-use-cache
-        (progn
-          (rails-routes--set-cache command-result)
-          (rails-routes--set-cache-validations t)))
+    (when rails-routes-use-cache
+      (rails-routes--set-cache command-result)
+      (rails-routes--set-cache-validations t))
 
     command-result))
 
 (defun rails-routes--get-routes-cached ()
-  "Get the cached routes if not exists."
+  "Get the routes, using the cache if possible."
   (let ((routes-result (if (cdr (assoc (projectile-project-name) rails-routes-cache-validations))
                            (cdr (assoc (projectile-project-name) rails-routes-cache))
                          (rails-routes--run-command))))
-    (if (eq routes-result nil) (rails-routes--run-command) routes-result)))
+    (if (eq routes-result nil)
+        (rails-routes--run-command)
+      routes-result)))
 
 (defun rails-routes--get-routes ()
-  "Handle if need to call cache or run directly the command."
-  (if rails-routes-use-cache (rails-routes--get-routes-cached) (rails-routes--run-command)))
+  "Get the routes, trying the cache first if `rails-routes-use-cache' is enabled."
+  (if rails-routes-use-cache
+      (rails-routes--get-routes-cached)
+    (rails-routes--run-command)))
 
 (defun rails-routes--guess-route (controller-full-path)
-  "Try to get the route name when rails routes don't show to us. CONTROLLER-FULL-PATH:  controller name plus action."
+  "Guess the route name from CONTROLLER-FULL-PATH.
+CONTROLLER-FULL-PATH is the controller name plus action."
   (let ((controller-path (nth 0 (split-string controller-full-path "#"))))
     (replace-regexp-in-string "\/" "_" controller-path)))
 
-(defun rails-routes--find (INSERT-CLASS)
-  "Ask for the route you want and insert on code.  INSERT-CLASS: if t, insert the prefix class."
+(defun rails-routes--insert (insert-class)
```

Comment:
the name of the parameter is different than the used on the body.  Change it to insert-CLASS.

Reply:
Yep, missed that. Fixed now.

Reply:
As a tip, you can suggest inline fixes using the suggestion button, then apply them directly.